README.md

```diff
@@ -173,6 +192,14 @@ Possible values of this variable:
 * `{state: absent}`: the ansible host name is unset in the insights-client config file and Host Based Inventory (HBI) is updated to use the system host name.
 * any other string value: the ansible host name is changed in Host Based Inventory (HBI).
 
+```yaml
+    rhc_inights:
```

Comment:
```suggestion
    rhc_insights:
```

Reply:
Fixed in commit [ff0199b](https://github.com/linux-system-roles/rhc/pull/177/commits/ff0199b1f95e5490a32844b1530f5b49f3a0167f)